client/components/email-verification/email-unverified-notice.jsx

```diff
@@ -131,7 +131,7 @@ export default class EmailUnverifiedNotice extends React.Component {
 				<p>
 					{ i18n.translate(
 						'To post and keep using WordPress.com you need to confirm your email address. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'To post and keep using WordPress.com you need to confirm your email address. Please click the link in the email we sent at %(email)s.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).